recipes/radosgw_civetweb.rb

```diff
@@ -0,0 +1 @@
+# Civetweb has no dependencies. It is included since the Firefly release.
```

Comment:
Rubocop doesn't like the lack of a final newline, but it should be good after it gets added :)